mitosheet/src/mito/components/toolbar/GraphTabs/LegendFormatOptions.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { GraphParamsBackend } from "../../..";
+import { RecursivePartial } from "../../../types";
+import Input from "../../elements/Input";
+import Toggle from "../../elements/Toggle";
+
+export const LegendFormatOptions = (props: {
+    params?: GraphParamsBackend;
+    updateGraphParam: (update: RecursivePartial<GraphParamsBackend>) => void;
+}) => {
+    return <div className='graph-element-style-options'>
+        <div className='horizontal-input-with-label'>
+            <p>X Position</p>
+            <Input
+                type='number'
+                value={`${props.params?.graph_styling?.legend.x ?? ''}`}
+                onChange={(e) => {
+                    props.updateGraphParam({
+                        graph_styling: {
+                            legend: {
+                                x: +e.target.value
+                            }
+                        }
+                    })
+                }}
+            />
+        </div>
+        <div className='horizontal-input-with-label'>
+            <p>Y Position</p>
```

Comment:
Also, setting the input type to number makes it very hard to write a negative number because when you write `-`, it just doesn't accept it. Also the up and down buttons probably enourages people to go outside of the -2 to 3 range. So, let's just use a regular text input and cast it. We do this elsewhere in filter inputs I believe.

Reply:
What if I just made it so that they can't go below -2 or above 3?

Reply:
I kind of like being able to use the up/down arrows to change the value

Reply:
Sure. Let's try it!